### DISCUSSION TITLE
this module vs resty

### DISCUSSION THREAD

**Discussion by isostatic** [OpenIDC/ngx_openidc_module#15] [Q&A]
I'm a bit confused over using this module or https://github.com/zmartzone/lua-resty-openidc

This feels nicer as it fits into stock nginx and is deployed by a simple deb rather than opm, but the lack of activity suggests the lua one is a better option?

**Comment by zandbelt** [comment]
the LUA one is more mature and has more features; this one is MVP, limited in its feature set but has native performance and commercial support/development
